core/src/main/java/hudson/model/Job.java

```diff
@@ -128,7 +128,7 @@
 import org.kohsuke.stapler.verb.POST;
 
 /**
- * A job is an runnable entity under the monitoring of Hudson.
+ * A job is a runnable entity under the monitoring of Hudson.
```

Comment:
Please revert. While technically correct, would prefer no unrelated cleanup in places not otherwise touched in the PR.

Reply:
Done